lib/firebase.dart

```diff
@@ -37,6 +39,17 @@ init() async {
       .collection(fb_constants.messagesCollection);
 }
 
+// Auth login and logout
+Future<firebase.UserCredential> googleLogin() async {
```

Comment:
For consistency, can you rename this to `googleSignIn`?

Reply:
Fixed